src/main/java/juja/sqlcmd/view/Console.java

```diff
@@ -0,0 +1,31 @@
+package juja.sqlcmd.view;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Scanner;
+
+public class Console implements View {
+
+    private Scanner scanner;
+    private OutputStream outputStream;
+
+    public Console(OutputStream printStream, InputStream inputStream) {
```

Comment:
Maybe is needed to rename printStream to outputStream?

Reply:
Yes of course! This is an artifact after refactoring :) Thanks!